MARC.js

```diff
@@ -244,6 +244,12 @@ record.prototype._associateDBField = function(item, fieldNo, part, fieldName, ex
 				
 				if(fieldName == "creator") {
 					item.creators.push(value);
+				} else if (fieldName == "ISBN" || fieldName == "abstractNote") {
```

Comment:
Well, the field 520 is repeatable and before we saved these all (?) in notes. I didn't wanted to make it  extract less than before (in theory). However, I haven't seen any multiple fields in abstract notes anyway.

Should I delete the abstractNote here again?